libkineto/src/output_json.h

```diff
@@ -91,4 +92,17 @@ class ChromeTraceLogger : public libkineto::ActivityLogger {
   std::ofstream traceOf_;
 };
 
+class ChromeTraceBaseTime {
+public:
+  ChromeTraceBaseTime() = default;
+  static void init() {
+    get();
+  }
+  static int64_t& get() {
+    static int64_t base_time = libkineto::timeSinceEpoch(
```

Comment:
This is triggering a CLANGTIDY warning:
```
Inlined static variable warning.
Local variable base_time with global storage is defined in inline function get.  Since this function is inlined, multiple copies of this variable may be used at runtime (due to dynamic linking) even though it has global storage.
Found local variables with global storage in inline functions which may
cause multiple copies to appear at runtime when used in dev mode (due
to dynamic linking and using -fvisibility-inlines-hidden). 
```

Reply:
Fixed this by making the class a static singleton